docs/content/preview/deploy/manual-deployment/verify-deployment.md

```diff
@@ -1,16 +1,29 @@
 ---
-title: Verify manual deployment
+title: Verify deployment
 headerTitle: Verify deployment
-linkTitle: 5. Verify deployment
+linkTitle: 4. Verify deployment
 description: How to verify the manual deployment of the YugabyteDB database cluster.
 menu:
   preview:
-    identifier: deploy-manual-deployment-verify-deployment
+    identifier: deploy-verify-2-manual
     parent: deploy-manual-deployment
     weight: 615
 type: docs
 ---
 
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="../verify-deployment-yugabyted/" class="nav-link">
+      yugabyted
+    </a>
+  </li>
+  <li >
+    <a href="../verify-deployment/" class="nav-link active">
+      Manual
+    </a>
+  </li>
+</ul>
+
```

Comment:
In the [Connect clients](https://deploy-preview-27433--infallible-bardeen-164bc9.netlify.app/preview/deploy/manual-deployment/verify-deployment/#connect-clients ) section, can the addresses be listed one after the other rather than in a long continuous line.